jina-embeddings/jina-embeddings-v2-base-en/config.yaml

```diff
@@ -0,0 +1,27 @@
+environment_variables: {}
+external_package_dirs: []
+hf_cache:
+  - repo_id: jinaai/jina-embeddings-v2-base-en
```

Comment:
you probably wanna pin the revision here just like you're doing in the load.

Reply:
Ah, makes sense, do you know how to do that? Would it just be `revision`: `abcd1234`?

Reply:
https://truss.baseten.co/reference/config#hf-cache-list-item-revision